fe/fe-core/src/main/java/org/apache/doris/analysis/FunctionCallExpr.java

```diff
@@ -246,7 +246,7 @@ private String paramsToSql(FunctionParams params) {
         if (params.isDistinct()) {
             sb.append("DISTINCT ");
         }
-        int len = params.exprs().size();
```

Comment:
Looks like we don't need `params` as the parameter of this method.
Because `fnParams` is a field of this class.